src/components/ui/button.tsx

```diff
@@ -5,24 +5,19 @@ import { cva, type VariantProps } from 'class-variance-authority';
 import { cn } from '@/lib/utils';
 
 const buttonVariants = cva(
-  'inline-flex items-center justify-center whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50',
+  'inline-flex items-center justify-center whitespace-nowrap rounded-xl text-lg font-normal ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 capitalize',
   {
     variants: {
       variant: {
-        default: 'bg-primary text-primary-foreground hover:opacity-80',
-        destructive:
-          'bg-destructive text-destructive-foreground hover:opacity-80',
-        outline:
-          'border border-input bg-background hover:bg-accent hover:text-accent-foreground',
-        secondary: 'bg-secondary text-secondary-foreground hover:opacity-80',
-        ghost: 'hover:bg-accent hover:text-accent-foreground',
-        link: 'text-primary underline-offset-4 hover:underline',
+        default: 'bg-primary text-white hover:opacity-80',
+        secondary:
+          'border border-primary bg-white hover:bg-slate-300 hover:opacity-50 hover:border-none ',
+        disable: 'bg-white border border-color-gray-500 text-gray-500',
+        destructive: 'bg-rose-600 text-white hover:opacity-80',
```

Comment:
The Text Button component needs to be added as well, can you please add here?
I think you can use [Link component](https://ui.shadcn.com/docs/components/button#link-1)

▼Figma
![Screenshot 2024-03-27 at 2 20 56 PM](https://github.com/Tascurator/tascurator-frontend/assets/132441773/3e153aae-205d-4122-b53c-e74d3fcd769d)

Reply:
I added the text button.